app/controllers/fines_controller.rb

```diff
@@ -31,4 +44,32 @@ def payments_response
   def payments
     (Hash.from_xml(Nokogiri::XML(payments_response).to_s) || {}).dig('LookupPatronInfoResponse', 'feeInfo')
   end
+
+  def fines_params
+    {
+      reason: fines.map(&:status).join(','),
+      billseq: fines.map(&:sequence).values_at(0, -1).join('-'),
+      amount: format('%.2f', fines.sum(&:owed))
+    }
+  end
+
+  def user_params
+    {
+      session_id: symphony_client.session_token,
+      group: patron.group? && 'G',
```

Comment:
This is forcing a user with a group to always pay a group fine? It seems like it should probably be contextual (based on the `params[:group]` value)?

Reply:
Now passing in a `group_choice` param to the `payments_helper`.

Reply:
Moved this to helper method